src/actions/receive-profile.js

```diff
@@ -123,7 +123,7 @@ export function loadProfile(
     // before finalizing profile view. That's why we are dispatching this action
     // after completing those steps inside `setupInitialUrlState`.
     if (initialLoad === false) {
-      await dispatch(finalizeProfileView(profile, config.geckoProfiler));
+      await dispatch(finalizeProfileView(config.geckoProfiler));
```

Comment:
We don't need to pass the profile here because we are dispatching the `PROFILE_LOADED` that sets the profile above this anyway.